satpy/tests/reader_tests/test_vaisala_gld360.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
This license is not consistent with the one above. IIRC we agreed to remove individual authors from the header.